compiler/one-cmds/one-codegen

```diff
@@ -94,15 +95,46 @@ def _get_parser(backends_list):
     return parser
 
 
-def _verify_arg(parser, args):
+def _verify_arg(parser, args, cfg_args):
     """verify given arguments"""
+    if oneutils.is_valid_attr(cfg_args, 'backend') and oneutils.is_valid_attr(
+            cfg_args, 'backends') and oneutils.is_valid_attr(cfg_args, 'command'):
+        parser.error(
+            '\'backend\' option and \'backends\' option cannot be used simultaneously.')
+
+    # Check if given backend from command line exists in the configuration file
+    if oneutils.is_valid_attr(args, 'backend') and oneutils.is_valid_attr(
+            cfg_args, 'backend'):
+        if args.backend != cfg_args.backend:
+            parser.error('Not found the command of given backend')
+
     # check if required arguments is given
     missing = []
-    if not oneutils.is_valid_attr(args, 'backend'):
+    if not oneutils.is_valid_attr(args, 'backend') and not oneutils.is_valid_attr(
```

Comment:
(optional) give `oneutils.is_valid_attr(args, 'backend')` others a flag. use the flags as to some human readable, meaningful function or variable. use this so that it is more understandable.